typespec/Applications.Core/containers.tsp

```diff
@@ -264,6 +264,30 @@ model Container {
   workingDir?: string;
 }
 
+@doc("Environment variables type")
+model EnvironmentVariable {
+  @doc("The value of the environment variable")
+  value?: string;
+
+  @doc("The reference to the variable")
+  valueFrom?: EnvironmentVariableReference;
+}
+
+@doc("The reference to the variable")
+model EnvironmentVariableReference {
+  @doc("The secret reference")
+  secretRef: EnvironmentVariableSecretReference;
+}
+
+@doc("The secret reference")
```

Comment:
Suggestion: We might be able to reuse an existing model SecretReference from Applications.Core with the same structure : https://github.com/radius-project/radius/blob/46bf036b9d3c4d18881a568845d71cf382d8f701/typespec/Applications.Core/environments.tsp#L177-L184

Reply:
resolved in offline discussions